Description: Drop -q and EMACS_UNIBYTE from startup script
 Removed -q switch from emacspeak.sh. The switch was causing problems
 in using "customize" command. On the other hand no-one could supply
 a reasonable explanation for the presence of this switch, here is
 the discussion link:
 http://www.cs.vassar.edu/~priestdo/emacspeak/2013/msg00548.html
 .
 Also removed EMACS_UNIBYTE flag, which is obsolete in emacs23 and up.
Author: Jarek Czekalski <jarekczek@poczta.onet.pl>
Bug-Debian: http://bugs.debian.org/730682
Forwarded: http://www.cs.vassar.edu/~priestdo/emacspeak/2013/msg00564.html
Reviewed-By: Paul Gevers <elbrus@debian.org>
Last-Update: 2013-11-29

--- a/etc/emacspeak.sh.def
+++ b/etc/emacspeak.sh.def
@@ -4,16 +4,11 @@
 # AKA  Use at your own risk, and if it breaks, you get to keep both pieces. 
 # Patches that are fully tested always welcome.
 #$Id$
-    if [ -f /etc/emacspeak.conf ]
+if [ -f /etc/emacspeak.conf ]
     then
     . /etc/emacspeak.conf
 fi
 
-if [ -f $HOME/.emacs ]
-then
-	INITSTR="-l $HOME/.emacs"
-fi
-
 CL_ALL=""
 for CL in $* ; do
 	if [ "$CL" = "-o" ]; then
@@ -28,14 +23,9 @@
 	elif [ "$CL" = "-d" ]; then
 		DTK_PROGRAM=dtk-soft
 		export DTK_PROGRAM
-	elif [ "$CL" = "-q" ]; then
-		INITSTR=""
 	else
 		CL_ALL="$CL_ALL $CL"
 	fi
 done
 
-
-EMACS_UNIBYTE=1
-export EMACS_UNIBYTE
-exec emacs -q -l <emacspeak-dir>/emacspeak-setup.el $INITSTR $CL_ALL
+exec emacs -l <emacspeak-dir>/emacspeak-setup.el $CL_ALL
